test/core/test/__snapshots__/jest-expect.test.ts.snap

```diff
@@ -220,6 +220,51 @@ exports[`asymmetric matcher error 15`] = `
 `;
 
 exports[`asymmetric matcher error 16`] = `
+{
+  "actual": "foo",
+  "diff": "- Expected: 
+toBeOneOf<bar,baz>
+
++ Received: 
+"foo"",
+  "expected": "toBeOneOf<bar,baz>",
+  "message": "expected 'foo' to deeply equal toBeOneOf<bar,baz>",
+}
+`;
+
+exports[`asymmetric matcher error 17`] = `
+{
+  "actual": "0",
+  "diff": "- Expected: 
+toBeOneOf<Any,,>
```

Comment:
The diff doesn't handle `null` and `undefined` correctly. It should be:

```suggestion
toBeOneOf<Any,null,undefined>
```

Reply:
Hmm, this is unfortunate. Maybe can we try `stringify` instead of `String`? :thinking: 

https://github.com/vitest-dev/vitest/blob/021944cd588958f3fa22d632e2bfd3efa0f1310d/packages/expect/src/jest-extend.ts#L134-L136

Reply:
Yes, this is better. But it also affects `toSatisfy()`